docs/device_automation_trigger.md

```diff
@@ -99,3 +100,18 @@ async def async_attach_trigger(hass, config, action, automation_info):
 ```
 
 The return value is a function that detaches the trigger.
+
+#### Add text and translations
+
+The *New Automation* user interface will display a human readable string in the device automation mapped to the event
+type.  Update `strings.json` and if you used script.scaffold then `translations/en.json` with the full set of supported event types:
+
```

Comment:
"New" is relative in time, so better to avoid that 😉 

```suggestion
The Automation user interface will display a human-readable string in the device automation mapped to the event
type.  Update `strings.json` and if you used `script.scaffold` then `translations/en.json` with the full set of supported event types:

```